app/kafka/manager/KafkaManagerActor.scala

```diff
@@ -87,6 +87,8 @@ object ClusterConfig {
             zkHosts: String,
             zkMaxRetry: Int = 100,
             jmxEnabled: Boolean,
+            jmxUser: Option[String] = Some(""),
```

Comment:
There should be no default values here, they should always be populated for user and pass.